### PR TITLE
 doc:ad717x_iio:Fixing Typos

### PR DESCRIPTION
**Pull request by D-Disha**

### CODE REVIEW

**Review by ribdp — APPROVED** [review]

**Comment by D-Disha** [comment]
Hi all, I have added ad7124 documentation to the same PR. Are there any comments on the same?

File: doc/sphinx/source/projects/ad7124_iio/ad7124_hardware_connection.png
Comment (jansunil) [inline doc/sphinx/source/projects/ad7124_iio/ad7124_hardware_connection.png:1]: Nucleo H5 supports only physical com port

Comment (jansunil) [inline doc/sphinx/source/projects/ad7124_iio/ad7124_hardware_connection.png:1]: Should we also add a mention of the DRDY GPIO pin that we are using?

**Comment by D-Disha** [comment]
Hi all, are there any more comments on this PR?

**Review by jansunil — APPROVED** [review]

**Review by SaikiranGudla — APPROVED** [review]

**Pull request merged**